src/Components/Book.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] KIndly ensure you dont disable linters but rather endeavor to fix your linter errors because linters force you to write clean code 👍🏽

Reply:
Done. Thank you